### DISCUSSION TITLE
[Request] Option to sync /saves/ to external source.

### DISCUSSION THREAD

**Discussion by Levistras** [OnionUI/Onion#1040] [Ideas]
Would be great if you could sync the saves folder externally (via ftp or samba) to a defined location so I could pick up either of my Mini+'s and have my saves and switcher the same.

Right now I can do this manually by having both online and connecting to them from PC but would be great if I could have the device check at startup or shutdown, or even if manually triggered, to grab latest or save latest versions to an external location.  A small version of cloud saves!

**Comment by Aemiii91** [comment]
Note, we're already working on a CloudSync solution here: https://github.com/OnionUI/Onion/pull/820
It's planned for 4.3

**Reply by Levistras** [reply]
Brilliant!  Thanks!

**Comment by poc-sm** [comment]
Hello 
It's a wonderful news !
but I've got a question
It seems you will add only backupfolder with rclone, am I right ? 
For exemple JELOS use two system and I think they are complementary 
1/ to backup folder (like saves) to a server with rclone (really useful for backup sometimes)
2/ provide syncthink buit-in server (really usefull to synchronise folder in real time with other device)  https://syncthing.net/
https://github.com/JustEnoughLinuxOS/distribution/blob/main/documentation/SETUP_SYNCTHING.md#syncthing-is-not-a-backup 

I think it can be great if both system can be include (rclone and syncthink)

**Reply by XK9274** [reply]
> Hello 
> It's a wonderful news !
> but I've got a question
> It seems you will add only backupfolder with rclone, am I right ? 
> For exemple JELOS use two system and I think they are complementary 
> 1/ to backup folder (like saves) to a server with rclone (really useful for backup sometimes)
> 2/ provide syncthink buit-in server (really usefull to synchronise folder in real time with other device)  https://syncthing.net/
> https://github.com/JustEnoughLinuxOS/distribution/blob/main/documentation/SETUP_SYNCTHING.md#syncthing-is-not-a-backup 
> 
> I think it can be great if both system can be include (rclone and syncthink) 

https://github.com/XK9274/syncthing-app-miyoo

**Comment by Rahhide** [comment]
check this links https://github.com/hotcereal/cloud-saves-miyoo-mini-plus

**Reply by Levistras** [reply]
Thanks, I'm already using both that and SyncThing on my MMP's.    Lately my issue has been getting Wifi to connect so I can use either... they're the only devices that have any issues in my house.

**Reply by bmix** [reply]
I wonder whether this becomes official. Would be nice. #820 seems closed.
